server/src/controllers/employer/getAllEmployers.js

```diff
@@ -16,12 +17,31 @@ const getAllEmployersRequestHandler = async (req, res) => {
       endDateAdded,
       ownerId,
       postalCode,
+      contactName,
     } = req.query;
 
     const query = {};
     if (employerName) {
       query.name = { [Op.like]: `%${employerName}%` };
     }
+
+    let employerContactIds = [];
+    if (contactName) {
+      const employerContacts = await EmployerContact.findAll({
+        where: {
+          name: { [Op.like]: `%${contactName}%` },
+        },
+      });
+      employerContactIds = employerContacts.map((contact) => contact.id);
+    }
+
+    logger.info(`Employer contact ids: ${employerContactIds}`);
+    logger.info(`Employer Name: ${contactName}`);
+
+    if (employerContactIds.length > 0) {
+      query.employer_contact_id = { [Op.in]: employerContactIds };
```

Comment:
I don't believe there is a field called employer_contact_id on employer. Instead here you can do 
`query.id = {[Op.in]: employerIds}` (Assuming you changed employerContactIds to employerIds as per my above comment)